taxi/urls.py

```diff
@@ -44,7 +47,24 @@
     path("cars/<int:pk>/delete/", CarDeleteView.as_view(), name="car-delete"),
     path("drivers/", DriverListView.as_view(), name="driver-list"),
     path(
-        "drivers/<int:pk>/", DriverDetailView.as_view(), name="driver-detail"
+        "drivers/<int:pk>/",
+        DriverDetailView.as_view(),
+        name="driver-detail"
+    ),
+    path(
+        "drivers/create/",
+        DriverCreateView.as_view(),
+        name="driver-create"
```

Comment:
Code style issue: According to the checklist, you should use `-` to split words in URL identification parameter `name`, not the `_`. So instead of `driver-create`, you should use `driver-create`.